pkg/remote/remote_test.go

```diff
@@ -817,3 +835,24 @@ func TestCreateDockerignoreFileWithFilesystem(t *testing.T) {
 		})
 	}
 }
+
+func TestGetOktetoPrefixEnvVars(t *testing.T) {
+	expectedEnvVars := map[string]string{
+		"OKTETO_ENV_VAR_1": "value1",
+		"OKTETO_ENV_VAR_2": "value2",
+	}
+
+	environ := []string{
+		"OKTETO_ENV_VAR_1=value1",
+		"OKTETO_ENV_VAR_2=value2",
+		"NON_OKTETO_ENV_VAR=value3",
+	}
+
+	prefixEnvVars := getOktetoPrefixEnvVars(environ)
+
+	for key, value := range expectedEnvVars {
+		if prefixEnvVars[key] != value {
```

Comment:
I would use `assert.Equal`.

Reply:
We are getting the `os.Environ` so any variable that is locally will be injected. I want to confirm that only the one ones I added in the test are the correct ones.

Reply:
you can still use 

```go
assert.Equal(t, value, prefixEnvVars[key])
```

and remove the if, no?